### PR TITLE
update re: APP-63

### PR DESCRIPTION
**Pull request by mauerbac**
@nickdirienzo does this make sense? Is it worth explaining an invalid library results in a 400 error?

### CODE REVIEW

File: src/content/rest/reference/projects/1-read.md [hunk 1]
```diff
@@ -10,7 +10,8 @@ fields:
   project_name: The name of the project in Optimizely
   project_status: Can be `Active` or `Archived`
   account_id: The account the project is associated with
-  include_jquery: Should be `true` or `false`
+  include_jquery: Should be `true` or `false`. If set to `false` the recommended version will be used.
```
Comment (nickdirienzo) [inline src/content/rest/reference/projects/1-read.md]: If set to `true`, the recommended jQuery version will be used, which is 1.11.3 trimmed.

File: src/content/rest/reference/projects/1-read.md [hunk 2]
```diff
@@ -10,7 +10,8 @@ fields:
...
+  library: The prefered Javascript library you would like to include. Must set `include_jquery` to `true`
```
Comment (nickdirienzo) [inline src/content/rest/reference/projects/1-read.md]: The preferred jQuery library you would like to use with your snippet. We support the following strings: "jquery-1.6.4-full", "jquery-1.6.4-trim", "jquery-1.11.3-full", and "jquery-1.11.3-trim". If you would like to not include jQuery, set this field to "none" and `include_jquery` to `false`.

File: src/content/rest/reference/projects/1-read.md (hunk 1 above)
Comment (mauerbac) [inline src/content/rest/reference/projects/1-read.md]: I'll fix the true mistake, but to stay consistent with the UI we should default to 1.6.4 trim -- can you change this?

Comment (nickdirienzo) [inline src/content/rest/reference/projects/1-read.md]: Our UI has 1.11.3 recommended, though?

screenshot 2016-04-28 12 07 53

Comment (mauerbac) [inline src/content/rest/reference/projects/1-read.md]: @bwll thoughts on this?

Comment (nickdirienzo) [inline src/content/rest/reference/projects/1-read.md]: For some clarity on the above, my project is set to use 1.6.4 full (for local testing), but the recommended version is 1.11.3 trimmed.

Comment (bwll) [inline src/content/rest/reference/projects/1-read.md]: @mauerbac 1.6.4 trim is the default when creating a new project through the UI, even though 1.11.3 is recommended

Comment (nickdirienzo) [inline src/content/rest/reference/projects/1-read.md]: @bwll I landed a change to upgrade all new projects going through the UI to 1.11.3. Is this a problem? Does this require an announcement to customers?

Comment (mauerbac) [inline src/content/rest/reference/projects/1-read.md]: @bwll @nickdirienzo so what's the finally consensus here? I'd love to push this out today

Comment (nickdirienzo) [inline src/content/rest/reference/projects/1-read.md]: 1.11.3 is officially our new default.

**Pull request merged**